content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,194 @@
+---
+title: April '21 Community Gems
+date: 2021-04-30
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running cml pipeline reports and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running cml pipeline reports  and more.
+picture: 2021-04-30/gems-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-community-gems/
+tags:
+  - Discord
+  - Gems
+  - DVC
+  - CML
+  - Remote Storage
+---
+
+Each month we go through our Discord messages to pull out some of the best
+questions from our community. AKA: Community Gems. 💎 This month we'd like to
+thank asraniel, Julian, mattlbeck, Ahti, **\_**, yikeqicn, lexzen, EdAb,
+FreshLettuce for inspiring this month's gems!
+
+As always, [join us in Discord](https://discord.com/invite/dvwXA2N) to get all
+your DVC and CML questions answered!
+
+## DVC
+
+### [Does the directory system have to already be set up before pushing to OwnCloud with a WebDav remote?](https://discord.com/channels/485586884165107732/485596304961962003/831472645508694046)
+
+A `dvc push` should set that up from your local drive. If you are receiving
+'failed to upload' and 'Remote resource not found' errors with your OwnCloud
+links and a `dvc doctor` report something like this,
+
+```dvc
+
+$ dvc doctor
+DVC version: 2.0.17 (pip)
+---------------------------------
+Platform: Python 3.8.5 on Linux-5.4.0-70-generic-x86_64-with-glibc2.29
+Supports: http, https, s3, webdav, webdavs
+Cache types: hardlink, symlink
+Cache directory: ext4 on /dev/sda8
+Caches: local
+Remotes: webdav
+Workspace directory: ext4 on /dev/sda8
+Repo: dvc, git
+```
+
+in this case you can use webdavs for the url and the ask_password parameter for
```

Comment:
We should expand on this.

- everything before this line is not that relevant to the problem and the solution in this case
- solution should start with something like (not literally, I'm writing this very fast. Would be great to clarify or even try this) "Most likely it means that something is wrong with the credentials/permissions setup ... Please make sure that you use the right protocol (`webdav` vs `webdavs`), and provide user name and/or token, password ,etc, etc ...."